src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial.mdx

```diff
@@ -0,0 +1,476 @@
+    SELECT name, duration
+    FROM Transaction
+    ```
+    
+  </Side>
+  <Side>
+  <img
+  title="Select and From with Duration query"
+  alt="A screenshot of a query with Select and From using Duration"
+  src={queriesnrqltutorial3}
+/>
+  </Side>
+</SideBySide>
+With this, you've completed queries that can return either all or some specific attributes of an event, as well as querying a specific number of events. Let's go to the next step: aggregating data.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="aggregate-data"
+    title="2. Aggregate data"
+  >
+<SideBySide>
+  <Side>
+Sometimes you may want to view summarized aggregate data across many events instead of viewing individual events. This is where NRDB shines. NRDB can scan billions of events and provide instant, real-time answers to questions about your data. For example, each `Transaction` event has a duration attribute that represents how many seconds the call took to execute. Let's see if we can discover the average duration across all `Transaction` events:
+```sql
+SELECT average(duration) 
+FROM Transaction
+```
+  </Side>
+  <Side>
+    <img
+  title="Select average duration from transaction query"
+  alt="A screenshot of a query selecting the average duration from transaction"
+  src={queriesnrqltutorial4}
+/>
+  </Side>
+</SideBySide>
+<SideBySide>
+  <Side>
+NRQL has many built-in functions you can use to aggregate event data. Commonly used functions include `max()`, `min()`,`average()`, `median()`, and `percentile()`. In the following query, we'll ask NRDB for the duration of the slowest transaction from the last 60 minutes To do this, we will ask NRDB for the `max(duration).`
+```sql
+SELECT max(duration) 
+FROM Transaction
+```
+  </Side>
+  <Side>
+    <img
+  title="Select maximum duration from transaction query"
+  alt="A screenshot of a query selecting the maximum duration from transaction"
+  src={queriesnrqltutorial5}
+/>
+  </Side>
+</SideBySide>
+<SideBySide>
+  <Side>
+We now know how long the slowest customer experience in the last 60 minutes took. Next, let's find the inverse and see how long the fastest experience was by asking NRDB for the `min(duration)`.
+```sql
+SELECT min(duration) 
+FROM Transaction
+```
+  </Side>
+  <Side>
+    <img
+  title="Select minimum duration from transaction query"
+  alt="A screenshot of a query selecting the minimum duration from transaction"
+  src={queriesnrqltutorial6}
+/>
+  </Side>
+</SideBySide>
+<SideBySide>
+  <Side>
+As you can see, `max()` and `min()` both return important data points. However, we may also want to perform arithmetic on our data. With 'sum()', we can do basic addition on a numeric attribute. In this query, we'll use a new attribute, `databaseCallCount`. This attribute counts the number of database calls each transaction makes. By adding them, we can get the total volume of database calls in the default 60 minute time window (this will be 0 if your apps don't talk to a database).
```

Comment:
```suggestion
As you can see, `max()` and `min()` both return important data points. However, we may also want to perform arithmetic on our data. With `sum()`, we can do basic addition on a numeric attribute. In this query, we'll use a new attribute, `databaseCallCount`. This attribute counts the number of database calls each transaction makes. By adding them, we can get the total volume of database calls in the default 60 minute time window (this will be 0 if your apps don't talk to a database).
```

Reply:
Thank you!